core/src/main/java/org/apache/iceberg/util/Tasks.java

```diff
@@ -145,7 +144,8 @@ public Builder(Iterable<I> items) {
       return this;
     }
 
-    public Builder<I> stopRetryOn(Class<? extends Exception>... exceptions) {
+    @SafeVarargs
```

Comment:
`SafeVarargs` requires the method to be `final`

Reply:
I wonder if we really should just ignore this warning, I'm trying to figure out if we really want this class to be extendable or not. If we start making the var-arg methods final are we making the class un-extendable? This is fine for now though, I think in particular this method wouldn't ever be extended.